packages/renderer/src/components/Editor/styles.css

```diff
@@ -0,0 +1,38 @@
+.Editor {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  width: 100%;
+  height: 100vh;
+}
+
+.Editor .header {
+  display: flex;
+  flex: 0 0 auto;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 0 16px;
+  height: 48px;
+}
+
+.Editor .header .back {
+  background-image: url("./back.svg");
```

Comment:
do we need this icon? I think we can use
`import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';`